### PR TITLE
support searchIterator && queryIterator

### PR DESCRIPTION
**Pull request by lentitude2tk**

### CODE REVIEW

File: examples/main/java/io/milvus/IteratorExample.java [hunk 1]
```diff
@@ -0,0 +1,361 @@
...
+        List<List<Float>> vectors = new ArrayList<>();
+        for (int n = 0; n < count; ++n) {
+            List<Float> vector = getFloatVector((float) n);
+            vectors.add(vector);
+        }
+
+        return vectors;
+    }
+
+    private static List<Float> getFloatVector(Float init) {
```
Comment (yhmo) [inline examples/main/java/io/milvus/IteratorExample.java]: There is already a common util method that can replace this method: https://github.com/milvus-io/milvus-sdk-java/blob/ab46ffcfdb17521998086f5fed7f76837677bbec/examples/main/java/io/milvus/CommonUtils.java#L36

File: examples/main/java/io/milvus/IteratorExample.java [hunk 2]
```diff
@@ -0,0 +1,361 @@
...
+            searchIteratorParamBuilder.withTopK(topK);
+        }
+
+        R<SearchIterator> response = milvusClient.searchIterator(searchIteratorParamBuilder.build());
+        handleResponseStatus(response);
+        return response.getData();
+    }
+
+
+    private List<List<Float>> generateFloatVectors(int count) {
```
Comment (yhmo) [inline examples/main/java/io/milvus/IteratorExample.java]: There is already a common util method that can replace this method:
https://github.com/milvus-io/milvus-sdk-java/blob/ab46ffcfdb17521998086f5fed7f76837677bbec/examples/main/java/io/milvus/CommonUtils.java#L45

File: examples/main/java/io/milvus/IteratorExample.java [hunk 3]
```diff
@@ -0,0 +1,361 @@
...
+    private static final Integer VECTOR_DIM = 8;
+    private static final String AGE_FIELD = "userAge";
+
+    private static final String INDEX_NAME = "userFaceIndex";
+    private static final IndexType INDEX_TYPE = IndexType.IVF_FLAT;
+    private static final String INDEX_PARAM = "{\"nlist\":128}";
+    private static final boolean CLEAR_EXIST = false;
+    private static final Integer NUM_ENTITIES = 1000;
+
+    private void handleResponseStatus(R<?> r) {
```
Comment (yhmo) [inline examples/main/java/io/milvus/IteratorExample.java]: There is a common util method can replace this method:
https://github.com/milvus-io/milvus-sdk-java/blob/ab46ffcfdb17521998086f5fed7f76837677bbec/examples/main/java/io/milvus/CommonUtils.java#L30

File: examples/main/java/io/milvus/IteratorExample.java (hunk 2 above)
Comment (lentitude2tk) [reply examples/main/java/io/milvus/IteratorExample.java]: After the subsequent synchronization of CommonUtils, then proceed with code simplification.

File: examples/main/java/io/milvus/IteratorExample.java (hunk 3 above)
Comment (lentitude2tk) [reply examples/main/java/io/milvus/IteratorExample.java]: After the subsequent synchronization of CommonUtils, then proceed with code simplification.

File: examples/main/java/io/milvus/IteratorExample.java (hunk 1 above)
Comment (lentitude2tk) [reply examples/main/java/io/milvus/IteratorExample.java]: After the subsequent synchronization of CommonUtils, then proceed with code simplification.

**Pull request merged**